webpack.config.composite-config-demo.js

```diff
@@ -0,0 +1,34 @@
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+module.exports = {
```

Comment:
You can use calypso build package to build so no need custom webpack and Babel configs here. See other /apps or /packages for examples or docs at calypso-build package.

Reply:
This file is just to run the demo. I don't think we have anything like this so far, and because the demo script is in the global package.json I thought maybe this should be here. But perhaps it should be a second webpack config inside the package?

I considered putting this demo in devdocs, but since this has no calypso dependencies, it seems like a poor experience to have to load and rebuild all of calypso just to develop this package. I asked in Slack and it was suggested that I create a custom script for the demo. The better solution would be a devdocs that does not depend on calypso. I started down that road but then was told about #33166 which I think is the better solution. However, we can't wait to work on this project until Storybook is available, so a custom dev web server seems like the best temporary solution.

Reply:
I also may not be understanding your comment correctly 😓 ... reading the `calypso-build` README doesn't fully clarify to me how I would use it to run this dev server.